crates/biome_js_analyze/tests/specs/nursery/useIterableCallbackReturn/invalid.js.snap

```diff
@@ -0,0 +1,1688 @@
+  
+    171 │ [].sort((a, b) => {});
+    172 │ [].sort(function(a, b) {});
+  > 173 │ [].toSorted((a, b) => {
+        │    ^^^^^^^^
+    174 │     return;
+    175 │ });
+  
+  i This return statement does not return a value.
+  
+    171 │ [].sort((a, b) => {});
+    172 │ [].sort(function(a, b) {});
+  > 173 │ [].toSorted((a, b) => {
+        │                        
+  > 174 │     return;
+        │     ^^^^^^
+    175 │ });
+    176 │ [].toSorted(function(a, b) {
+  
+
+```
+
+```
+invalid.js:176:4 lint/nursery/useIterableCallbackReturn ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i This callback passed to toSorted() iterable method should always return a value.
+  
+    174 │     return;
+    175 │ });
+  > 176 │ [].toSorted(function(a, b) {
+        │    ^^^^^^^^
+    177 │     return;
+    178 │ });
+  
+  i This return statement does not return a value.
+  
+    174 │     return;
+    175 │ });
+  > 176 │ [].toSorted(function(a, b) {
+        │                             
+  > 177 │     return;
+        │     ^^^^^^
+    178 │ });
+    179 │ [].toSorted((a, b) => {
+  
+
+```
+
+```
+invalid.js:179:4 lint/nursery/useIterableCallbackReturn ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i This callback passed to toSorted() iterable method should always return a value.
+  
+    177 │     return;
+    178 │ });
+  > 179 │ [].toSorted((a, b) => {
+        │    ^^^^^^^^
+    180 │     if (a > b) {
+    181 │         return;
+  
+  i This return statement does not return a value.
+  
+    180 │     if (a > b) {
+    181 │         return;
+  > 182 │     } else if (a < b) {
+        │                        
+  > 183 │         return;
+        │         ^^^^^^
+    184 │     } else {
+    185 │         return 1;
+  
+  i This return statement does not return a value.
+  
+    178 │ });
+    179 │ [].toSorted((a, b) => {
+  > 180 │     if (a > b) {
+        │                 
+  > 181 │         return;
+        │         ^^^^^^
+    182 │     } else if (a < b) {
+    183 │         return;
+  
+
+```
+
+```
+invalid.js:188:4 lint/nursery/useIterableCallbackReturn ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i This callback passed to toSorted() iterable method should always return a value.
+  
+    186 │     }
+    187 │ });
+  > 188 │ [].toSorted((a, b) => {
+        │    ^^^^^^^^
+    189 │     if (a > b) {
+    190 │         return;
+  
+  i This return statement does not return a value.
+  
+    189 │     if (a > b) {
+    190 │         return;
+  > 191 │     } else if (a < b) {
+        │                        
+  > 192 │         return;
+        │         ^^^^^^
+    193 │     }
+    194 │ });
+  
+  i This return statement does not return a value.
+  
+    187 │ });
+    188 │ [].toSorted((a, b) => {
+  > 189 │     if (a > b) {
+        │                 
+  > 190 │         return;
+        │         ^^^^^^
+    191 │     } else if (a < b) {
+    192 │         return;
+  
+  i This callback does not return a value.
```

Comment:
This message feels redundant, because we already said at the beginning that, and we have two additional code frames that show that.